CMakeLists.txt

```diff
@@ -132,6 +132,10 @@ target_link_libraries(astra_list_devices astra_wrapper)
 
 add_executable(astra_test_wrapper test/test_wrapper.cpp )
 target_link_libraries(astra_test_wrapper astra_wrapper ${Boost_LIBRARIES})
+
+add_executable(action_image_client test/action_image_client.cpp)
```

Comment:
why did you add the image client to the test folder?

Reply:
Because that is the client that publishes images to debug_images topic in order to test rgb/grab_images action. Where would you put it?

Reply:
Where you usually add node executables... In my brain in test you add only tests... but maybe it's just my brain :)